servers/v3-proxy-api/src/routes/v3Get.integration.ts

```diff
@@ -1719,4 +1719,66 @@ describe('v3Get', () => {
       },
     );
   });
+  describe('Charset Handler', () => {
+    it('rewrites bad utf8', async () => {
+      const requestSpy = jest.spyOn(
+        GraphQLCalls,
+        'callSavedItemsByOffsetComplete' as const,
+      );
+      jest
+        .spyOn(GraphQLClient.prototype, 'request')
+        .mockResolvedValueOnce(mockGraphGetComplete);
+      const response = await request(app)
+        .post('/v3/get')
+        .send({
+          consumer_key: 'test',
+          access_token: 'test',
+          since: 1712766000,
+          detailType: 'complete',
+        })
+        .set('Content-Type', 'application/json; charset=UTF8');
+      expect(response.body).toEqual(expectedGetComplete);
+      expect(requestSpy).toHaveBeenCalledTimes(1);
+      expect(requestSpy.mock.calls[0][3]).toMatchObject({
```

Comment:
this checks that the request is made (makes through the try/catch) but not that the utf8 header is rewritten. Can you check the headers on the request if they're forwarded?

Reply:
@kschelonka  Content-Type gets deleted on forwarding, this is mainly on express accepting the value.